analyze_bugbot_report.py

```
@@ -0,0 +1,92 @@
+    
+    # Files with security issues
+    security_files = []
+    for filename, issues in file_issues.items():
+        for issue in issues:
+            if issue['severity'] in ['CRITICAL', 'HIGH']:
+                security_files.append((filename, issue))
+                break
+    
+    if security_files:
+        print("\n⚠️  Files with Security Issues (CRITICAL/HIGH):")
+        for filename, issue in security_files:
```

Ruff (0.11.9)

Loop control variable `issue` not used within loop body

(B007)